### DISCUSSION TITLE
Can't Install Git Desktop on Win11 (.NET Framework issue)

### DISCUSSION THREAD

**Discussion by cragnar002** [community/community#36252] [Other Feature Feedback, Questions, & Ideas]
I'm trying to install Git Desktop and I get an error that I need the following version of the .NET Framework: v4.0.30319. In the Registry Editor, I have version 4.8.04161 installed. Why isn't it installing? I've tried reinstalling .NET before with no solutions yet.

**Comment by BlasioGodi** [comment]
Have you already installed the latest Windows Updates? Trying checking for the Cumulative updates and any other Windows may have and check whether this fixes your issue. The updates are always accompanied by the necessary .NET Framework update.

**Reply by cragnar002** [reply]
Only thing I see is 22H2, which I heard bricks a lot of drivers. Up to date otherwise.

**Comment by jfullstackdev** [comment]
never tried that on Windows 11 but I installed GitHub Desktop for several times on Windows 10 without any special setup, it's always successful 

maybe this will help, particularly the one in AppData folder

https://stackoverflow.com/questions/71848629/github-desktop-does-not-start-any-more

**Reply by cragnar002** [reply]
It's weird since I have Git Desktop on my other machine that _has_ Win11. I'll give this link a try.

**Comment by ossanna16** [comment]
Hey there @cragnar002 and thank you for asking a good question! I'm glad you found our community 🙂
